mobilebot/mobilebot.c

```diff
@@ -138,8 +146,7 @@ void read_mb_sensors(){
     mb_state.last_yaw = mb_state.tb_angles[2];
     mb_state.tb_angles[2] = imu_data.dmp_TaitBryan[TB_YAW_Z];
     mb_state.temp = imu_data.temp;
-    mb_state.yaw_delta = mb_angle_diff_radians(mb_state.last_yaw, mb_state.tb_angles[2]);
-    mb_state.gyro_heading_delta = mb_state.gyro[2] * PI / (180.0 * SAMPLE_RATE_HZ);
+    mb_state.yaw_delta = mb_state.tb_angles[2] - mb_state.last_yaw;
```

Comment:
Same as other comment, I think we want tb_angles[3] yaw (heading)

If i'm not mistaken, the tait-bryan anges we get are in Roll, Pitch, Yaw order

Reply:
I think you're correct, roll-1, pitch-2, yaw-3 is usually the standard.

Reply:
as Xun pointed out, [2] does refer to the 3rd item (my bad!)

Reply:
Yup, I've been using too much Matlab...